src/styles/blocks/content.scss

```diff
@@ -0,0 +1,9 @@
+.main {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
```

Comment:
check layout when in row less than 4 cards.
for fix it do not set `justify-content: space between` and add right margin for 
![Screenshot from 2023-04-17 21-11-08](https://user-images.githubusercontent.com/33029864/232574290-c03a27b6-b8c3-4db4-b98f-08c26dcddc09.png)
card (except last in the row)